README.md

```diff
@@ -10,21 +10,20 @@
 3. Невозможность собрать метрики.
 
 ## Состав:
-Backend:  
-Flask
+### Backend:  
+FastAPI
 
-Frontend:  
+### Frontend:  
 Jinja2
```

Comment:
Not really. Jinja2 is a templating engine for your Python code. Your real frontend would be generated HTML + CSS + JS files.